doc/man3/OSSL_CORE_MAKE_FUNC.pod

```diff
@@ -0,0 +1,34 @@
+=pod
+
+=head1 NAME
+
+OSSL_CORE_MAKE_FUNC - OpenSSL reserved symbols
+
+=head1 SYNOPSIS
+
+ #include <openssl/core_dispatch.h>
+
+ #define OSSL_CORE_MAKE_FUNC(type,name,args)
+
+=head1 DESCRIPTION
+
+There are certain macros that may appear in OpenSSL header files that are
+reserved for internal use. They should not be used by applications or assumed
+to exist.
+
+All the macros listed in the synopsis above are reserved.
+
```

Comment:
Added in section?

Reply:
I don't understand what change you are suggesting here?

Reply:
Do you need a HISTORY section?

Reply:
Fixed.

Reply:
Fixed.